src/schema/v2/artwork/index.ts

```diff
@@ -370,6 +377,53 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         description:
           'Represents the "**medium type**", such as _Painting_. (This field is also commonly referred to as just "medium", but should not be confused with the artwork attribute called `medium`.)',
       },
+      collectionsConnection: {
+        type: CollectionsConnectionType,
+        args: pageable({
+          page: { type: GraphQLInt },
+          size: { type: GraphQLInt },
+          default: { type: GraphQLBoolean },
+          saves: { type: GraphQLBoolean },
+          sort: { type: CollectionSorts },
+        }),
+        resolve: async (parent, args, context, _info) => {
+          const { id: artwork_id } = parent
+          const { collectionsLoader, userID: meID } = context
+          if (!collectionsLoader) return null
+
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          const { body, headers } = await collectionsLoader({
+            artwork_id,
+            user_id: meID,
+            private: true,
+            default: args.default,
+            saves: args.saves,
+            sort: args.sort,
+            size,
+            offset,
+            total_count: true,
+          })
+          const bodyWithMe = (body ?? []).map((obj) => {
+            return { ...obj, userID: meID }
```

Comment:
**question**: why do we need to return `userID` here? Sorry if this is a stupid question 😅

Reply:
aaaah, got it
<img width="1533" alt="Insomnia 2023-03-01 16-40-44" src="https://user-images.githubusercontent.com/3513494/222155246-c6156381-e765-4942-b8b5-40aed994d16f.png">